### ISSUE TITLE
Add button is not working in "Add Ranges" in opennms

### ISSUE THREAD

**Issue by vishleshpp** [opennms-forge/docker-horizon-core-web#27]
In docker-horizon-core-web:bleeding docker image, "Add" button is non-responsive in opennms->admin->Configure opennms->Run Single Discovery Scan->Include Ranges->Add New, while adding ip range for Discovery.

**Comment by stevecheckoway** [comment]
I believe this is due to a JavaScript error.

```
Uncaught TypeError: this.parse is not a function
    at Object.l [as Address4] (ipv4.js:45)
    at checkIpRange (ipaddress-js.js:19)
    at doAddIncludeRange (add-ir.jsp?mode=scan:131)
    at HTMLButtonElement.onclick (add-ir.jsp?mode=scan:221)
l @ ipv4.js:45
checkIpRange @ ipaddress-js.js:19
doAddIncludeRange @ add-ir.jsp?mode=scan:131
onclick @ add-ir.jsp?mode=scan:221
```

**Comment by indigo423** [comment]
Thank you very much for reporting this bug and for additional work to investigate what the problem is. I've migrated your findings to our OpenNMS internal issue tracker to get this problem tracked in further releases. The issue is migrated to here: https://issues.opennms.org/browse/NMS-10135

**Comment by mvrueden** [comment]
May I ask, which browser you are using? I am not able to reproduce the issue.

**Comment by stevecheckoway** [comment]
@mvrueden I'm using Google Chrome Version 66.0.3359.139 on macOS.

**Comment by mvrueden** [comment]
Yeah, it was probably introduced with the `core/web-assets` refactor we did a couple of month back. @indigo423 was able to reproduce it. Thank you for reporting.

**Comment by indigo423** [comment]
@stevecheckoway @vishleshpp our devs were able to fix this issue which will be published with Horizon 22.0.0 release on 17th May 2018. Thanks a lot for reporting and diagnosing this problem.
